### PR TITLE
Servlet

### PR DESCRIPTION
**Pull request by azhara-a**
Rename functions and variables in dark-mode-switch for readability.

DataServlet now returns a list of fun facts as JSON. Script.js fetches the facts and places them on the page as an unordered list

### CODE REVIEW

File: portfolio/pom.xml [hunk 1]
```diff
@@ -23,6 +23,11 @@
       <version>4.0.1</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>com.google.code.gson</groupId>
+      <artifactId>gson</artifactId>
+  <version>2.8.6</version>
+</dependency>
```
Comment (lminchillo) [inline portfolio/pom.xml:30]: nit: fix indentation

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -15,6 +15,7 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
+import com.google.gson.Gson;
```
Comment (lminchillo) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:18]: nit: fix import order

You can check out style guides here: https://google.github.io/styleguide/

File: portfolio/pom.xml (hunk 1 above)
Comment (azhara-a) [reply portfolio/pom.xml]: Fixed in the next commit

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 2 above)
Comment (azhara-a) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:17]: Fixed in the next commit

File: portfolio/src/main/webapp/dark-mode.css [hunk 3]
```diff
@@ -25,3 +25,9 @@
     border-color: rgba(0, 0, 0, 0.3);
     background-color: #333;
 }
+
+[data-theme="dark"] .form-control {
+    border: rgb(56, 56, 56);
+    background-color: rgb(56, 56, 56);
+    color: rgba(238, 238, 238, 0.8);
+}
```
Comment (lminchillo) [inline portfolio/src/main/webapp/dark-mode.css:33]: Cool!

**Pull request merged**